Source/VolunteerReporting/Read/CaseReportsForListing/CaseReportForListing.cs

```diff
@@ -2,39 +2,39 @@
 using System;
 using System.Collections.Generic;
 using System.Runtime.Serialization;
+using Dolittle.ReadModels;
 
 namespace Read.CaseReportsForListing
 {
-    public class CaseReportForListing
+    public class CaseReportForListing :  IReadModel, IHaveReadModelIdOf<Guid>
     {
-        
-        //TODO: Comment woksin: This whole Read project for casereports needs a rework like what I did for UserManagement
-        public Guid Id { get; private set; }
-        public CaseReportStatus Status { get; internal set; }
-        public DataCollectorId DataCollectorId { get; internal set; }
-        public string DataCollectorDisplayName { get; internal set; } = "Unknown"; //TODO: Handle localization for default values or handle in view
-        public HealthRiskId HealthRiskId { get; internal set; }
-        public string HealthRisk { get; internal set; } = "Unknown"; //TODO: Handle localization for default values or handle in view
-        public string Message { get; internal set; }
-        //TODO: Remove custom get logic once obsolete properties can be removed
-        public int NumberOfMalesUnder5 { get; internal set; }
-        public int NumberOfMalesAged5AndOlder { get; internal set; }
-        public int NumberOfFemalesUnder5 { get; internal set; }    
-        public int NumberOfFemalesAged5AndOlder { get; internal set; }
-        public DateTimeOffset Timestamp { get; internal set; }
-        public Location Location { get; internal set; }
-        public string Origin { get; internal set; }
-        public IEnumerable<string> ParsingErrorMessage { get; internal set; }
+        public Guid Id { get; set; }
+        public CaseReportStatus Status { get; set; }
+        public DataCollectorId DataCollectorId { get; set; }
+        public string DataCollectorDisplayName { get; set; } = "Unknown"; //TODO: Handle localization for default values or handle in view
+        public HealthRiskId HealthRiskId { get; set; }
+        public string HealthRisk { get; set; } = "Unknown"; //TODO: Handle localization for default values or handle in view
+        public string Message { get; set; }
+        public int NumberOfMalesUnder5 { get; set; }
+        public int NumberOfMalesAged5AndOlder { get; set; }
+        public int NumberOfFemalesUnder5 { get; set; }    
+        public int NumberOfFemalesAged5AndOlder { get; set; }
+        public DateTimeOffset Timestamp { get; set; }
+        public Location Location { get; set; }
+        public string Origin { get; set; }
+        public IEnumerable<string> ParsingErrorMessage { get; set; }
+
         //TODO: Remove properties below from model once data is migrated to new property names or can be removed
         //This feels dirty. Waiting for input from @einari on how to handle backwardscompability better
         [Obsolete]
-        public int? NumberOfFemalesAgedOver4 { get; internal set; }
```

Comment:
If these are [obselete] we can take them out. We don't have a previous version of the software running.